Content: Cats Watch Fox Hunt in Winter on Plains

Artyphex opened the pull request:
The patrol spots a fox leaping and diving into the snow. They either try it themselves, laugh, or snap at each other to focus.

Artyphex committed:
Content: Cats Watch Fox Hunt in Winter on Plains

The patrol spots a fox leaping and diving into the snow. They either try it themselves, laugh, or snap at each other to focus.

Artyphex committed:
Content: New Loner Names

Added Aziraphale, Crowley, Karlach, and Fenris as loner name possibilities

Thlumyn commented on resources/dicts/names/names.json:
If you want to add more names, please make that a separate PR

Artyphex replied:
Sorry about this! I didn't realize if I made another without branching first it would attach itself to the first PR. Won't happen again!

Thlumyn commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -1574,5 +1574,87 @@
+            "chance_of_success": 30,
+            "relationship_constraint": null,
+            "pl_skill_constraint": null,
+            "intro_text": "Leaf-bare is taking its toll. Your patrol is cold, frusterated, and most of all, hungry. r_c notices something odd. Is that a fox in the distance? Why is it jumping into the snow like that?",
```
spelling - "frustrated"

Artyphex replied:
Tweaked in source file, will update soon.

Thlumyn commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -1574,5 +1574,87 @@
+            "relationship_constraint": null,
+            "pl_skill_constraint": null,
+            "intro_text": "Leaf-bare is taking its toll. Your patrol is cold, frusterated, and most of all, hungry. r_c notices something odd. Is that a fox in the distance? Why is it jumping into the snow like that?",
+            "decline_text": "First the cold, and the snow, and now a fox? This is too much for one day, p_l decides. Back to camp, another patrol can try again later. ",
```
is this meant to be p_l or r_c ? This might be a different cat than the one that saw the fox

Artyphex replied:
In my mind, I thought the implication was the patrol was speaking to each other so if the cats were different it wouldn't be confusing. Do you think it would be better if it were all just s_c?

Thlumyn commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -1574,5 +1574,87 @@
+            "decline_text": "First the cold, and the snow, and now a fox? This is too much for one day, p_l decides. Back to camp, another patrol can try again later. ",
+            "success_outcomes": [
+                { 
+            "text": "s_c watches, copying how the fox walks close to the snow, ears down, imagining what it might be listening for until- that's it! s_c leaps into the air, diving headfirst into the snow. The patrol snickers as {PRONOUN/s_c/poss} rear flails above the snow, but that snickering turns to praise when s_c pops back up with a fat mouse between {PRONOUN/s_c/poss} teeth.",
```
again, which cat is this supposed to be? r_c or s_c ?

Artyphex replied:
Same as above! Do you think them being potentially different cats (r_c sees it and s_c reacts) is too confusing?

Thlumyn commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -1574,5 +1574,87 @@
+                    "amount": 5
+                },
+            { 
+            "text": "The whole patrol stares at the sight of a fox, something cats fear, stuck upside with its whole rump flailing up and undignified. r_c can't stop the unattractive snort that leaves {PRONOUN/r_c/poss}, and then the whole patrol is laughing. For this one moment, your cats forget about the cold in the fur and hunger in their bellies to enjoy a moment with their friends.",
```
{PRONOUN/r_c/poss} mouth? Their what?

Artyphex committed:
Content: Cats Watch Fox Hunt in Winter on Plains - Update

Switched some r_c's to s_c's to make the text less confusing, but left it where I thought it still fit. Fixed a spelling error and some wording.

Thlumyn commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -1574,5 +1574,87 @@
+                    "amount": 5
+                },
+            { 
+            "text": "The whole patrol stares at the sight of a fox, something cats fear, stuck upside with its whole rear flailing up and undignified. r_c can't stop (PRONOUN/r_c/self) from snorting at the sight. Then the whole patrol is laughing. For this one moment, your cats forget about the cold in the fur and hunger in their bellies to enjoy a moment with their friends.",
```
Looks like this pronoun tag should have {} instead of ()

Thlumyn commented on resources/dicts/names/names.json:
The changes to names still need to be removed

Thlumyn commented:
Just as a heads up, to make things easier you'll probably want to do your changes like this on a separate branch on your personal fork of this: like Artyphex/catfoxwinter or something like that, instead of Artyphex/development. It makes it easier to sync changes with the base clangen repository, and also easier to compare changes with the clangen/development branch (so its not, like, comparing "development" to "development").

Artyphex commented:
> Just as a heads up, to make things easier you'll probably want to do your changes like this on a separate branch on your personal fork of this: like Artyphex/catfoxwinter or something like that, instead of Artyphex/development. It makes it easier to sync changes with the base clangen repository, and also easier to compare changes with the clangen/development branch (so its not, like, comparing "development" to "development").

Got it! I'll do that in the future. Still figuring Github out, I appreciate your patience!

Artyphex committed:
Content: Cats Watch Fox Hunt in Winter on Plains - Update 2

Fixed a brackets error

Artyphex commented:
I messed this PR up, closing it and reopening a fixed one
